app/src/main/java/org/fossasia/susi/ai/data/SkillDetailsModel.kt

```diff
@@ -74,6 +77,27 @@ class SkillDetailsModel : ISkillDetailsModel {
         })
     }
 
+    override fun sendReport(query: ReportSkillQuery, listener: ISkillDetailsModel.OnReportSendListener) {
+        val map: MutableMap<String, String> = HashMap()
+        map.put("access_token", query.accessToken)
+        map.put("feedback", query.feedback)
+        map.put("skill", query.skill)
+        map.put("group", query.group)
+        map.put("model", query.model)
+        reportSkillResponseCall = ClientBuilder.getSusiApi().reportSkill(map)
```

Comment:
Please create this map inside a static function in ClientBuilder.java and create the map there. For a reference, you can check the implementation of fetch feedback feature.

Reply:
@arundhati24 I was asked to follow the pattern of how the feedback is send, and to pass that map values using data class. Okay, now I will move that function in ClientBuilder.java.